bot_impl.py

```diff
@@ -3411,7 +3411,9 @@ async def generate_possibilities(text, people):
     possibilities = []
     for person in people:
         if (
-            person.nick is not None and text.lower() in person.nick.lower()
+                person.nick is not None and text.lower() in person.nick.lower()
+        ) or (
+                person.global_name is not None and text.lower() in person.global_name.lower()
```

Comment:
works when selecting from members but not when selecting from players.
Player needs to shadow the Member global_name

review https://github.com/fiveofswords/botc-bot/pull/53